### PR TITLE
(node/lsstcam-quad.cp.lsst.org) deploy host as el9

### PR DESCRIPTION
**Pull request by dtapiacl**

### CODE REVIEW

**Review by glennmorris — APPROVED** [review]

File: hieradata/node/lsstcam-quad.cp.lsst.org.yaml
Comment (jhoblitt) [inline hieradata/node/lsstcam-quad.cp.lsst.org.yaml:1]: Please add this device to ipam (a device and its fqdn ip).

File: spec/hosts/nodes/lsstcam-quad.cp.lsst.org_spec.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        it { expect(nm_keyfile['ipv4']['dns-search']).to eq('cp.lsst.org;') }
+      end
+
+      context 'with enp3s0' do
+        let(:interface) { 'enp3s0' }
+
+        it_behaves_like 'nm enabled interface'
+        it_behaves_like 'nm ethernet interface'
+        it { expect(nm_keyfile['ipv4']['address1']).to eq('192.168.1.126/24') }
+        it { expect(nm_keyfile['ipv4']['method']).to eq('manual') }
```
Comment (jhoblitt) [inline spec/hosts/nodes/lsstcam-quad.cp.lsst.org_spec.rb]:
```suggestion
        it_behaves_like 'nm manual interface'
```

**Review by jhoblitt — CHANGES_REQUESTED** [review]

File: hieradata/node/lsstcam-quad.cp.lsst.org.yaml
Comment (dtapiacl) [reply hieradata/node/lsstcam-quad.cp.lsst.org.yaml:1]: Done, created a new device type for this board.

**Review by jhoblitt — APPROVED** [review]
The device and both static IPs have been confirmed as being present in ipam.

File: hieradata/node/lsstcam-quad.cp.lsst.org.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    content:
+      connection:
+        id: "enp3s0"
+        uuid: "edb4ebb1-6d05-427b-b6ea-d483bdf32ec7"
+        type: "ethernet"
+        interface-name: "enp3s0"
+      ethernet: {}
+      ipv4:
+        method: "manual"
+        address1: "192.168.1.120/24"
```
Comment (jhoblitt) [inline hieradata/node/lsstcam-quad.cp.lsst.org.yaml:29]: I missed it at first glance, this IP isn't associated with a host either.

**Review by jhoblitt — CHANGES_REQUESTED** [review]

File: hieradata/node/lsstcam-quad.cp.lsst.org.yaml (hunk 2 above)
Comment (dtapiacl) [reply hieradata/node/lsstcam-quad.cp.lsst.org.yaml:29]: done

**Review by jhoblitt — APPROVED** [review]
LGTM. Thank you updating ipam.

**Review by jhoblitt — CHANGES_REQUESTED** [review]
Please squash 3b868586af01c0dd8eb075dd6bb7a682505fa71a

**Comment by dtapiacl** [comment]
> Please squash [3b86858](https://github.com/lsst-it/lsst-control/commit/3b868586af01c0dd8eb075dd6bb7a682505fa71a)

Commit was squashed on to: [cf05f59](https://github.com/lsst-it/lsst-control/pull/1149/commits/cf05f59be7e8de2ad705d76757b8e3a22990e376)

**Comment by csilva-cl** [comment]
> Please squash [3b86858](https://github.com/lsst-it/lsst-control/commit/3b868586af01c0dd8eb075dd6bb7a682505fa71a)

why squashing? perhaps a more descriptive commit message is better?

**Review by jhoblitt — APPROVED** [review]

**Pull request merged**